workflow.src

```diff
@@ -0,0 +1,121 @@
+{
+  "title":  "Workflow for editors and other contributors"
+}
+---
+<main>
+      <p>
+        A good practice for GitHub is to use <a href="https://help.github.com/articles/about-pull-requests/">pull requests</a>.
+        It lets you tell others about changes you've pushed to the repository.
+        Once a pull request is opened, you can discuss and review the potential changes with
+        others and add follow-up commits before the changes are merged into the repository.
+      </p>
+  <ol>
+    <li>
+      <p>
+        Navigate to the repo.
+      </p>
+      <pre><code>cd specs-on-github</code></pre>
+    </li>
+    <li>
+      <p>
+        Checkout the main branch of your repository (in general, this is the <code>master</code> branch).
+      </p>
+      <pre><code>git checkout master</code></pre>
+    </li>
+    <li>
+      <p>
+        Make sure the main branch is up-to-date with upstream.
+      </p>
+      <pre><code>git pull</code></pre>
+    </li>
+    <li>
+      <p>
+        Create a new branch for your upcoming pull request (one branch per pull request). Please make branch names
+        informative - by including the issue or bug number for example.
+      </p>
+      <pre><code>git checkout -B my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Make your edits in your favorite editor.
+      </p>
+    </li>
+    <li>
+      <p>
+        Add your edits to prepare your commit by staging them. Best is to use the interactive mode (New files still
+        need to be added explictly).
+      </p>
+      <pre><code>git add -p</code></pre>
+      <p>If you ignore an edit, it will be reproposed to you the next time you run the command. This allow you to make
+      multiple edits at once and split them into different more meaningfull/easy-to-review commits.</p>
+    </li>
+    <li>
+      <p>
+        Commit your staged edits. If you're fixing an issue, reference it in the commit e.g. <code>#87</code>
```

Comment:
Editors should really do this in an --amend. But I guess this is ok as start.

Reply:
also, to close an issue with a commit, it's not enough to mention it - you need to explicitly include "close #xx" or "fix #xx"

Reply:
How about: git commit --amend -m "&lt;purpose of your edits> (fix #number)"